modules/mediawiki/files/bin/deploy-mediawiki.py

```diff
@@ -33,12 +33,34 @@ def run(args, start):
         os.system(f'/usr/local/bin/logsalmsg {text}')
     else:
         print(text)
+    if args.world:
+        if args.pull:
+            pull = str(args.pull).split(',')
+            if 'world' not in pull:
+                pull.append('world')
+                args.pull = ','.join(pull)
```

Comment:
doesn't seem needed

Reply:
How come? This is to set args.pull to append world if it doesn't exist already, if world is being ran, so that the following git pull commands are used for it.

Reply:
but after this point you use the pull variable

Reply:
Yeah, but only if world is used is pull already set, if it isn't pull isn't set, and that's why pull is redefined below, if it's redefined below without args.pull being set which is what this is for, then pull being used here has no effect has it'd be overwritten with args.pull.